src/Router.tsx

```diff
@@ -49,7 +49,10 @@ const WRAPPER_STYLES: CSSProperties = {
 }
 
 const INNER_STYLES: CSSProperties = { width: '500px', height: '500px' }
-
+// route and used in query enabling
+export const ROUTES = {
+  GAMMA: '/gamma'
```

Comment:
why is this the only route abstracted to a variable? are you wanting to separate routes using tanstack vs not?